lib/src/geojson.dart

```diff
@@ -176,14 +211,38 @@ class Position extends CoordinateType {
   Position.of(List<num> list)
       : assert(list.length >= 2 && list.length <= 3),
         super(list);
+
   factory Position.fromJson(List<num> list) => Position.of(list);
 
-  // TODO implement override operators +, -, * with vector operations
+  Position operator +(Position p) => Position.of([
+        lng + p.lng,
+        lat + p.lat,
```

Comment:
does wraparound get handled in another location?

Reply:
probably in the `Position.of` constructor, but I can't quite follow the call chain here in the github UI

Reply:
yeah, I'm not sure if the logic belongs here or not, it sort of depends on the use cases. Is there a use case for getting positions that are technically out of bounds or should the value we return from _any_ function always be normalized to within the ±90deg/±180deg range?

Reply:
Let's move this to a separate discussion, because it would – as you pointed out – affect a lot of different parts of the library